testing/functional_tests/test_compare_models.py

```diff
@@ -1,6 +1,7 @@
 import logging
 import os
-from functional_tests.t_utils import remove_tmp_dir, __tmp_dir__, create_tmp_dir, __data_testing_dir__
+from testing.functional_tests.t_utils import __tmp_dir__, create_tmp_dir, __data_testing_dir__
+from testing.common_testing_util import remove_tmp_dir
```

Comment:
`def test_compare_models():` needs the `get_functional_test_files` fixture.

Reply:
Thanks for spotting that! Added.